d2l-button-icon.html

```diff
@@ -98,7 +105,10 @@
 			:host([translucent]) button:focus {
 				border: none;
 				background-color: var(--d2l-color-celestine);
-				box-shadow: none;
+			}
+
+			:host([translucent]) button:focus {
+				box-shadow: inset 0px 0px 0px 2px var(--d2l-color-celestine), inset 0px 0px 0px 3px white;
```

Comment:
Don't this and the previous rule have the same selector?

Reply:
The same selector is there, but it's grouped with the others to share the border and background properties.

Reply:
Oops yep, sorry -- the limited lines GitHub showed me were misleading!